code/modules/paperwork/photocopier.dm

```diff
@@ -120,7 +120,7 @@
 		else if(doccopy)
 			for(var/i = 0, i < copies, i++)
 				if(toner > 5 && !busy && doccopy)
-					new /obj/item/documents/photocopy(src, doccopy)
+					new /obj/item/documents/photocopy(loc)

```

Comment:
You remind me of some newb who went "I SPOTTED INFINITE CARGO POINTS EXPLOIT, BUY POINTS AND COPY MANIFESTS" in issues here. Ha ha ha. Read the fucking code.